src/main/resources/avro/references.avdl

```diff
@@ -31,65 +31,43 @@ field. New client and server implementations should prefer the "graph" mode.
 record Reference {
 
   /**
-  The `Segment` describing this `Reference`'s sequence in the `ReferenceSet`'s
+  The `Segment` describing this `Reference`'s `Sequence` in the `ReferenceSet`'s
   sequence graph.
 
-  The `Segment` will generally cover the entirety of the sequence, but may be
-  shorter if only a subrange of a sequence is to be included in the
+  The `Segment` will generally cover the entirety of the `Sequence`, but may be
```

Comment:
OK, I'm going to pull `Segment` out of `Reference` so it only lives in `Path`. You might now have to define your `Reference`s in the forward orientation on your `Sequence`s.